src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -59,7 +61,14 @@ public void Dispose()
         public void Loaded(ViewLoadedParams p)
         {
             LoadedParams = p;
-            PythonDependencies = new GraphPythonDependencies(LoadedParams.CurrentWorkspaceModel, LoadedParams.StartupParams.CustomNodeManager);
+
+            var ironPythonVersion = new Version(3, 0, 0);
+            if(LoadedParams.StartupParams.Preferences is PreferenceSettings prefs)
+            {
+                 Version.TryParse(prefs.IronPythonResolveTargetVersion,out ironPythonVersion);
```

Comment:
Don't we have Internal visible set for PythonMigrationViewExtension.  Can you access the internal field?

Reply:
Not a big deal...  This works too.